src/pages/messages/TasksPage.tsx

```diff
@@ -153,13 +153,16 @@ const TasksPage = () => {
     .filter(Boolean)
     .join(' AND ');
 
+  const getViewingScopeFromUrlOrDefaultToUsersViewingScope =
```

Comment:
Bloated name to try to specify why this is done

Reply:
I prefer bloated names over comments that eventually lies.